trillian_log_api.proto

```diff
@@ -82,9 +82,9 @@ service TrillianLog {
   }
 
   // Returns the latest signed log root for a given tree. Corresponds to the
-  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  The server will
-  // return InvalidArgument if first_tree_size is greater than the
-  // LatestSignedLogRoot available to that server.
+  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  If
+  // first_tree_size > LatestSignedLogRoot, the server will return rpc code
+  // Unavailable with the latest SignedLogRoot attached in the status details.
```

Comment:
"attached in the status details" how?  There should be enough detail here to allow someone implementing a client (in a language other than Go) to write code to get at the details.

Reply:
amended to `status.Details()` -  https://godoc.org/google.golang.org/grpc/status#Status.Details 

The gRPC Status object is a first class citizen in all languages that gRPC supports. Is there something more specific here that you have in mind?